<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appoinment</class>
 <widget class="QDialog" name="appoinment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton{
	font-size: 15px;
font-family: roboto,sans-serif;
font-weight: 800;	
}

QLabel{
	font-size:20px;
	font-weight:bold;
}

QPushButton{
color:black;
font-weight:bold;
font-size:13px;
background-color:transparent;
border:2px solid #8b0000;
border-radius:15px;

}

QPushButton:hover{
color:white;
background:#8b0000;
border:2px solid white;
border-radius:15px;

}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>09:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>10:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>11:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>610</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>APPOINT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Morning Slot :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>190</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>12:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Noon Slot :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="n1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>15:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="n2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>16:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="n3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>340</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>17:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="n4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>340</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>19:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Night Slot :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>510</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>20:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>21:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>510</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>22:00</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>510</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>23:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="at">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at1">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at4">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at7">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at8">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at10">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at11">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>470</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="at12">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Availibility</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
